plugins/loading/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { Plugin } from '@rematch/core';
+export interface LoadingConfig {
+    name?: string;
+    whitelist?: string[];
+    blacklist?: string[];
+    asNumber?: boolean;
+}
+declare const _default: (config?: LoadingConfig) => Plugin<import("@rematch/core").Models, import("@rematch/core").Action<any, any>>;
```

Comment:
I didn't know you could import in a generic. Solves that problem.

Unfortunate we can't supply the correct typings to actions.

This is definitely an improvement!

Reply:
@ShMcK No better way to find it, but at least TypeScript doesn't report an error.